jetstream/tests/integration/test_analysis_integration.py

```diff
@@ -30,11 +31,24 @@ def build_query_test_project(instance, *args, **kwargs):
             query = query.replace("telemetry", static_dataset)
             return query
 
+        orig_cluster = dask.distributed.LocalCluster.__init__
+
+        def mock_local_cluster(instance, dashboard_address, processes, threads_per_worker):
+            return orig_cluster(
+                instance,
+                dashboard_address=dashboard_address,
+                processes=False,
+                threads_per_worker=threads_per_worker,
+            )
+
```

Comment:
Why don't we want to test in multiprocess mode?

Reply:
pytest is running into issues here with mocking. If multiple processes are used then `build_query_test_project` get ignored resulting in an invalid query being executed. I'm not entirely sure why, but the easiest fix was to not use multiple processes for tests.

Reply:
Oh, yeah, thanks for explaining. Subprocesses start with clean state so they won't see mocks. This means that if we're accidentally relying on process state for real, this test won't catch it -- we _could_ consider providing some flag or environment variable that tells child processes that they should modify the query somehow, but I'm hoping that's unlikely since dask is managing all of the serialization for us and I think we're well-behaved, so we shouldn't need to block this.

Reply:
Ah, that makes sense. Yeah, keeping it as is should be fine. Introducing environment variables that change the test behaviour also make things harder to reason about.